run_pypolychord.py

```diff
@@ -38,28 +37,32 @@ def prior(hypercube):
 def dumper(live, dead, logweights, logZ, logZerr):
     print("Last dead point:", dead[-1])
 
+#| Create a paramnames file
+paramnames = [('p%i' % i, r'\theta_%i' % i) for i in range(nDims)]
+paramnames += [('r*', 'r')]
+
```

Comment:
This should be moved down to the anesthetic portion, and loaded in as a `columns` keyword.